pkg/api/testapi/testapi.go

```diff
@@ -268,15 +266,6 @@ func init() {
 			externalTypes:        api.Scheme.KnownTypes(externalGroupVersion),
 		}
 	}
-	if _, ok := Groups[kubeadm.GroupName]; !ok {
```

Comment:
And yet, that's where we arrived.  How frustrating.  Alright, I'll unwind the fuzzing tests either today or early next week.